third_party/libprotobuf_mutator.BUILD

```diff
@@ -0,0 +1,30 @@
+# Copyright 2021 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+licenses(["notice"])
+
+cc_library(
+    name = "libprotobuf_mutator",
+    srcs = glob(
+        ["src/*.cc", "src/libfuzzer/*.cc"], 
```

Comment:
I'm pretty surprised to see that they don't already have a BUILD file of their own. Would it work to just use `**/*.cc` for `srcs` and `**/*.h` for headers? It could make this file more resistant to changes in directory structure.

Reply:
It could, it could make it less too (if something incompatible gets added)... I think I'd like to leave it as is for now.